RunoPW/dist/metadata.json

```diff
@@ -14,7 +14,7 @@
   },
   "url": "runo.pw",
   "version": "2.0.0",
-  "logo": "https://i.imgur.com/t06m0ly.png",
+  "logo": "https://i.imgur.com/utrvCGI.png",
   "thumbnail": "https://i.imgur.com/zMfRQH9.png",

```

Comment:
Logo is ok now,
but you still need to remove that ugly white line at the bottom of the thumbnail